hello-neon/app/src/main/cpp/helloneon.c

```diff
@@ -114,7 +114,7 @@ jstring Java_com_example_helloneon_HelloNeon_stringFromJNI(JNIEnv* env,
   strlcat(buffer, "Neon version   : ", sizeof buffer);
 
   family = android_getCpuFamily();
-  if ((family != ANDROID_CPU_FAMILY_ARM) &&
+  if ((family != ANDROID_CPU_FAMILY_ARM) && (family!=ANDROID_CPU_FAMILY_ARM64) &&
```

Comment:
I'm pretty sure this also works for x86_64, and rather than continuing to extend this list, you should instead just remove this block. I'm not sure what rv64 plans to do about this, but I'll talk with them whenever the time comes for that.